velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -140,6 +140,14 @@ class StringTest : public SparkFunctionBaseTest {
     return evaluateOnce<std::string>("left(c0, c1)", str, length);
   }
 
+  std::optional<std::string> substring_index(
+      const std::optional<std::string>& str,
```

Comment:
no need to handle nulls; remove optional

Reply:
Removed.